src/.vuepress/config.js

```diff
@@ -51,12 +51,12 @@ module.exports = {
         link: "/hello-minimal/",
       },
       {
-        text: "Quickstart",
+        text: "Getting Started",
         link: "/quickstart/",
       },
       {
-        text: "Tutorial",
-        link: "/tutorial/",
+        text: "Tutorials",
+        link: "/tutorials/",
```

Comment:
This needs to be changed back Tutorial or change the folder name in source to tutorials